src/page-modules/assistant/details/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { ButtonLink } from '@atb/components/button';
+import { TripPatternWithDetails } from '../server/journey-planner/validators';
+import { PageText, useTranslation } from '@atb/translations';
+import { MonoIcon } from '@atb/components/icon';
+import { Typo } from '@atb/components/typography';
+
+import style from './details.module.css';
+
+export type AssistantDetailsProps = {
+  tripPattern: TripPatternWithDetails;
+};
+
+export function AssistantDetails({ tripPattern }: AssistantDetailsProps) {
+  const { t } = useTranslation();
+  const fromName = tripPattern.legs[0].fromPlace.name;
+  const toName = tripPattern.legs[tripPattern.legs.length - 1].toPlace.name;
+  return (
+    <div className={style.container}>
+      <div className={style.headerContainer}>
+        <ButtonLink
+          mode="transparent"
+          href="/assistant" // @TODO: Use correct href.
```

Comment:
I think this can be `href` should be supplemented with a onClick that does `history.back()` and `e.preventDefault()`.

```suggestion
          href="/assistant"
          onClick={(e) => {
            e.preventDefault();
            history.back();
          }
```

This a classic example of progressive enhancement. You can cmd + click to open new window for back, but also actually do back navigation to pop state.

Reply:
That's neat! I didn't know one could have both 💡